lib/modules/manager/gomod/extract.spec.ts

```diff
@@ -94,5 +94,39 @@ replace (
         ],
       });
     });
+
+    it('extracts digest updates for untagged releases', () => {
+      const goMod = `
```

Comment:
use the new codeblock template helper.

Reply:
will remove this test completely as it's already covered by the existing test